pyproject.toml

```diff
@@ -3,8 +3,18 @@ requires = ["hatchling"]
 build-backend = "hatchling.build"
 
 [tool.hatch.build]
```

Comment:
I believe the default build backend is now uv-build - do we still need the tool.hatch sections?